plugins/win-wasapi/win-wasapi.cpp

```diff
@@ -1138,17 +1138,42 @@ bool WASAPISource::ProcessCaptureData()
 			return false;
 		}
 
+		// ToDo figure out if we need to reset capture here
```

Comment:
Is this comment leftover? Also the OBS single-line style tends to be `/* Comment */`

Reply:
Leftover, I'll remove it.